src/components/button-toggle-group/__spec__.js

```diff
@@ -141,10 +153,34 @@ describe('ButtonToggleGroup', () => {
       });
     });
 
-    it('is decorated with a validation if a error is present', () => {
-      wrapper.setState({ errorMessage: 'Error', valid: false });
-      const errorDiv = wrapper.find('.common-input__message--error');
-      expect(errorDiv.prop('children')).toEqual('Error');
+    
+    describe('when has input-validation error ', () => {
+      beforeEach(() => {
+        jest.useFakeTimers();
+        wrapper = mount(
+          <ButtonToggleGroup
+            label={ 'Label' }
+            value={ 'foo' }
+          >
+            <ButtonToggle>Foo</ButtonToggle>
+          </ButtonToggleGroup>
+        );
+      });
+  
+      afterEach(() => {
```

Comment:
Could the three lines around Timers be called once in a top level beforeEach?

Reply:
Yes I should have tidied this up. It's done now